effect/default_effector.go

```diff
@@ -73,3 +73,24 @@ func (e *DefaultEffector) MergeEffects(expr string, effects []Effect, results []
 
 	return result, nil
 }
+
+// IsHit return whether it should be explain
+func (e *DefaultEffector) IsHit(expr string, eft Effect) (bool, error) {
```

Comment:
Why not use `MergeEffects`? Add `IsHit` is breaking changes.

Reply:
`results` is used to store matcher results in the case of float64.

If users use special matcher and their own designed effects, It may be used. I know this situation will be rare, but if we add feature to it, will this mislead the user？

Reply:
Unused the `results` in Casbin. So we don't need to care it.

Use `MergeEffects`：
```golang
e.eft.MergeEffects(e.model["e"]["e"].Value, policyEffects, nil)
```

Reply:
Thanks for your guidance, use `MergeEffects()` in the new commit

Reply:
Latest implementation from https://github.com/casbin/casbin/issues/355#issuecomment-615056049.  

For your previous implementation, you just need to change `isHit, err := e.eft.MergeEffects(e.model["e"]["e"].Value, []effect.Effect{policyEffects[i]}, nil)`

Reply:
Now, look good to me.